components/ILIAS/Chatroom/chat/package.json

```diff
@@ -3,10 +3,9 @@
   "description": "ILIAS CHAT",
   "version": "2.0.0",
   "dependencies": {
-    "async": "^3.2",
+    "async": "3.2",
```

Comment:
As decided by the JF, we should use v. `3.x` for `winston`, see: https://github.com/ILIAS-eLearning/ILIAS/pull/6769#issuecomment-1850061501.

We already tested this successfully (based on 9.x and an upgraded library and chat component) with v. `^3.13.0`